manifest/parse.go

```diff
@@ -40,8 +40,13 @@ func (m metadata) String() string {
 	if len(sp) > 1 {
 		apiBase = strings.Join(sp[:len(sp)-1], "/")
 	}
-
-	return fmt.Sprintf("%s, %s, %s (%s)", m.Metadata.Namespace, m.Metadata.Name, m.Kind, apiBase)
+	name := m.Metadata.Name
+	if a := m.Metadata.Annotations; a != nil {
+		if baseName, ok := a["helm.sh/base-name"]; ok {
+			name = baseName
+		}
+	}
+	return fmt.Sprintf("%s, %s, %s (%s)", m.Metadata.Namespace, name, m.Kind, apiBase)
```

Comment:
Does it still show the diff on `metadata.name`? Then it looks all good! If you could also add a test case to show how the output looks like when you diffed two secrets or configmaps, that would be great.

Reply:
Yes, it does! I'll find some time in the near future will add the test.

Reply:
And done.